src/components/Apps/AccountListTable/const.js

```diff
@@ -283,7 +283,7 @@ export const accountQuickFilters = (vm) => [
       {
         label: 'Web',
         filter: {
-          category: 'website'
+          category: 'web'
         }
       },
       {
```

Comment:
The code you provided seems to be incomplete with an unclear context. It's difficult for me to provide a meaningful analysis without knowing what the original code is supposed to do or which programming languages it is written in. Could you please elaborate on this?